components/release/kustomization.yaml

```diff
@@ -11,9 +11,18 @@ kind: Kustomization
 images:
 - name: quay.io/redhat-appstudio/release-service
   newName: quay.io/redhat-appstudio/release-service
-  newTag: ddac1272d2cea70a7b21ca0d6f65bc0ed9dbbd67
+  newTag: 0009b2836e62f86e53d40f86c101e8c790c71036
 
 namespace: release-service
 
-patches:
+patchesStrategicMerge:
```

Comment:
This part isn't needed. We removed the `namePrefix` piece in the release-service repo that we only added for kcp compatibility so the name issue for the service won't be there anymore

Reply:
updated

Reply:
returned the patches section